lib/fizzy/limits.hpp

```diff
@@ -13,8 +13,8 @@ constexpr uint32_t PageSize = 65536;
 
 /// The maximum memory page limit as defined by the specification.
 /// It is only possible to address 4 GB (32-bit) of memory.
-constexpr uint32_t MemoryPagesValidationLimit = (4 * 1024 * 1024 * 1024ULL) / PageSize;
-static_assert(MemoryPagesValidationLimit == 65536);
+constexpr uint32_t MemoryPagesMaxLimit = (4 * 1024 * 1024 * 1024ULL) / PageSize;
```

Comment:
Sorry for being late to the party. I think we should make the "max" and the "default" variable following the same scheme, either `MemoryPagesMaxLimit`/`MemoryPagesDefaultLimit`, or `MaxMemoryPagesLimit`/`DefaultMemoryPagesLimit`.

@chfast @gumb0 what do you think?

Reply:
Sounds good, I would change to `MaxMemoryPagesLimit`/`DefaultMemoryPagesLimit`.